module/pom.xml

```diff
@@ -256,7 +285,15 @@
       <version>${mule.version}</version>
       <scope>test</scope>
     </dependency>
-
+    <dependency>
+      <groupId>org.mule.sdk</groupId>
+      <artifactId>mule-sdk-api</artifactId>
+      <version>0.7.0-rc12</version>
+    </dependency>
+    <dependency>
+      <groupId>com.sun.activation</groupId>
+      <artifactId>jakarta.activation</artifactId>
```

Comment:
Is the version coming from the bom in dependency management?

Reply:
yes, but it's not necessary anymore